### PR TITLE
new file:   packages/linwrap/linwrap.9.0.3/opam

### PR DESCRIPTION
**Pull request by UnixJunkie**

### CODE REVIEW

**Comment by mseri** [comment]
There seems to be some conflict related to dolog:
```
# Detecting depexts using vars: arch=x86_64, os=linux, os-distribution=debian, os-family=debian
[ERROR] No solution for linwrap.9.0.3: The following dependencies couldn't be met:
          - linwrap -> dolog >= 6.0.0
          - linwrap -> molenc -> dolog < 5.0.0
        Your request can't be satisfied:
          - No available version of dolog satisfies the constraints
```
or
```
[ERROR] No solution for linwrap.9.0.3: The following dependencies couldn't be met:
          - linwrap -> dolog >= 6.0.0
          - linwrap -> molenc -> dolog < 5.0.0 -> ocaml < 4.06.0
              base of this switch (use `--unlock-base' to force)
        Your request can't be satisfied:
          - No available version of dolog satisfies the constraints
```

File: packages/linwrap/linwrap.9.0.3/opam [hunk 1]
```diff
@@ -0,0 +1,85 @@
+opam-version: "2.0"
+authors: "Francois Berenger"
+maintainer: "unixjunkie@sdf.org"
+homepage: "https://github.com/UnixJunkie/linwrap"
+bug-reports: "https://github.com/UnixJunkie/linwrap/issues"
+dev-repo: "git+https://github.com/UnixJunkie/linwrap.git"
+license: "BSD-3"
```
Comment (mseri) [inline packages/linwrap/linwrap.9.0.3/opam:7]:
```suggestion
license: "BSD-3-Clause"
```

Is ⬆️ the correct one? (see https://spdx.org/licenses/)

Comment (UnixJunkie) [reply packages/linwrap/linwrap.9.0.3/opam:7]: I guess so

**Comment by UnixJunkie** [comment]
Ok, I will release an updated version of molenc after this one is accepted.

**Comment by mseri** [comment]
Since this depends on molenc, would it be possible to release molenc, then rebase this and have it go through the testing instead? Right now this seems uninstallable everywhere anyways

**Comment by UnixJunkie** [comment]
ah, ok, sorry. I will release the latest molenc first, then.

**Comment by UnixJunkie** [comment]
will be resubmited after the latest molenc has been accepted

**Pull request closed without merging**